execution/executor-utils/Cargo.toml

```diff
@@ -25,5 +25,8 @@ libra-workspace-hack = { path = "../../common/workspace-hack", version = "0.1.0"
 libra-crypto = { path = "../../crypto/crypto", version = "0.1.0" }
 libra-types = { path = "../../types", version = "0.1.0" }
 
+[dev-dependencies]
+libra-types = { path = "../../types", version = "0.1.0", features = ["fuzzing"] }
```

Comment:
why not add the features flag to line 26?

Reply:
Because line 24 is commented out with a '#', and isn't the start of a dev dependencies section.

Reply:
This will cause feature unification. You need to enable this from the testing feature below than then pass the feature explicitly or have dependent packages turn it on.

Reply:
good point, thanks!   was tired when getting these to compile.